### DISCUSSION TITLE
Wrong Altitude

### DISCUSSION THREAD

**Discussion by Two-Rivers-Obs** [Stellarium/stellarium#3665] [General]
Hey all, I have an issue. I have a custom landscape in 23.4. The issue I have is that the horizon is about 10 degrees higher than it should be. If I load the Zero landscape, it is right, but with my landscape, it is way off.
![custom](https://github.com/Stellarium/stellarium/assets/125199482/7ac0cc25-aff7-453a-938a-fabe27f06244)
![zero](https://github.com/Stellarium/stellarium/assets/125199482/4f560805-9fbd-46f8-9990-f3b76fe4f3a3)

**Comment by gzotti** [comment]
Then you have to fix your panorama, obviously, which is out of scope of Stellarium. Does the panorama cover 360°x180°? If you just want to press down the horizon for a quick visual fix disregarding accuracy, just configure maptex_top. See the User Guide for details.

**Reply by Two-Rivers-Obs** [reply]
Thank you for the quick reply. I will give that a try.

**Reply by Two-Rivers-Obs** [reply]
I did adjust it and all is fine now. Thank you.
